gui/wxpython/location_wizard/wizard.py

```diff
@@ -2675,19 +2683,17 @@ def OnWizFinished(self):
 
     def CreateProj4String(self):
         """Constract PROJ.4 string"""
-        location = self.startpage.location
         proj = self.projpage.p4proj
-        projdesc = self.projpage.projdesc
         proj4params = self.paramspage.p4projparams
 
-#        datum = self.datumpage.datum
-        if self.datumpage.datumdesc:
-            datumdesc = self.datumpage.datumdesc + ' - ' + self.datumpage.ellipse
-        else:
-            datumdesc = ''
+#       datum = self.datumpage.datum
+#        if self.datumpage.datumdesc:
+#            datumdesc = self.datumpage.datumdesc + ' - ' + self.datumpage.ellipse
+#        else:
+#            datumdesc = ''
+
```

Comment:
I don't see a reason why this code should be kept there and there is no comment saying that either, so I would say just delete that.